src/Repository/ScheduledContentRepositoryTrait.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+namespace Zicht\Bundle\PageBundle\Repository;
+
+trait ScheduledContentRepositoryTrait
+{
+    private function applyPublishedWhereClausesToQueryBuilder(QueryBuilder $qb, string $alias = 'p'): QueryBuilder
```

Comment:
Goed dat je dit oppakt!

In PgoSupport hebben we ook een nog iets mooiere variant gemaakt... even kijken waar die is gebleven...

Reply:
Zeker, `Criteria` altijd :+1: